.env

```diff
@@ -1,4 +1,7 @@
-APP_DB_URL=jdbc:postgresql://127.0.0.1:5432/jpc?user=jpc&password=letmein
+APP_DB_URL="jdbc:postgresql://127.0.0.1:5432/jpc?user=jpc&password=letmein"
```

Comment:
These db changes are for integrating with Flyway and ShedLock.  A migration creates the locking table used by Shedlock.